kubernetes/types.go

```diff
@@ -426,6 +426,7 @@ type IstioObject interface {
 	SetObjectMeta(meta_v1.ObjectMeta)
 	DeepCopyIstioObject() IstioObject
 	HasWorkloadSelectorLabels() bool
+	HasMatchLabelsSelector() bool
```

Comment:
Not sure if this should belong to the type IstioObject interface, but I don't want to hold this one, this can be revisit later.